[Bug]: gimp build failed

### Problem description

[gimp.txt](https://github.com/termux/termux-packages/files/9187126/gimp.txt)


### What steps will reproduce the bug?

build pkg

### What is the expected behavior?

_No response_

### System information

termux-info:

```

```

https://github.com/2096779623/termux-packages/runs/7514768094?check_suite_focus=true

https://github.com/2096779623/termux-packages/blob/fix-issues-11357/x11-packages/gimp/build.sh

```bash

(glib-compile-resources:13177): GLib-GIO-CRITICAL **: 06:53:39.006: g_subprocess_wait: assertion 'G_IS_SUBPROCESS (subprocess)' failed

(glib-compile-resources:13177): GLib-GObject-CRITICAL **: 06:53:39.006: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
gimp-color-picker-cursors.gresource.xml: Failed to close file descriptor for child process (Operation not permitted).

(glib-compile-resources:13176): GLib-GIO-CRITICAL **: 06:53:39.007: g_subprocess_wait: assertion 'G_IS_SUBPROCESS (subprocess)' failed

(glib-compile-resources:13176): GLib-GObject-CRITICAL **: 06:53:39.007: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
gimp-tool-cursors.gresource.xml: Failed to close file descriptor for child process (Operation not permitted).
make[2]: *** [Makefile:937: gimp-tool-cursors.c] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: *** [Makefile:958: gimp-color-picker-cursors.c] Error 1
make[2]: Leaving directory '/home/builder/.termux-build/gimp/build/cursors'
make[1]: *** [Makefile:856: all-recursive] Error 1
make[1]: Leaving directory '/home/builder/.termux-build/gimp/build'
make: *** [Makefile:757: all] Error 2
Error: Process completed with exit code 2.
```

https://gist.github.com/nathabonfim59/b088db8752673e1e7acace8806390242

Why ???🤔

Is it a Docker problem?

--- a/x11-packages/gimp/build.sh
+++ b/x11-packages/gimp/build.sh
@@ -15,7 +15,7 @@ HOST_GLIB_COMPILE_RESOURCES=glib-compile-resources
 "
 
 termux_step_pre_configure() {
-	LDFLAGS+=" -landroid-shmem"
+	LDFLAGS+=" -landroid-shmem -lm"
 
 	local pc_dir=$TERMUX_PKG_BUILDDIR/_pkgconfig
 	mkdir -p $pc_dir

